src/test/videopress/edit.js

```diff
@@ -117,6 +120,44 @@ describe( "Update VideoPress block's settings", () => {
 		} );
 	} );
 
+	/*
+	 * RATING SETTINGS
+	 * Loop through each of the possible ratings and select each one
+	 */
+	RATING_OPTIONS.forEach( ( option, index ) => {
+		// Skip the default setting, as it is already selected
+		if ( index === 0 ) return;
```

Comment:
Another reason behind skipping the default setting is that, out-of-the-box, it leads to a `Found multiple elements with text: G` error on Android. This is due to the fact the setting is already selected and used as a label for the picker component. I don't feel it's necessary to add test coverage for this case (selecting an already selected setting) so have opted for the most straightforward approach of skipping it. I'd be curious if others have different opinions on this.

Reply:
> Another reason behind skipping the default setting is that, out-of-the-box, it leads to a Found multiple elements with text: G error on Android. This is due to the fact the setting is already selected and used as a label for the picker component.

Maybe for this case, we could query the text using the [`within` helper](https://callstack.github.io/react-native-testing-library/docs/api#within-getqueriesforelement) to limit the scope to the `Privacy and Rating` screen.

> I don't feel it's necessary to add test coverage for this case (selecting an already selected setting) so have opted for the most straightforward approach of skipping it. I'd be curious if others have different opinions on this.

I agree it's not necessary to test selecting the already selected setting. The only part that could be tested is that after a selection is made, the option is displayed on the bottom sheet. Although, I think this logic should have been already tested by the unit tests of the `SelectControl` component.

Reply:
> Maybe for this case, we could query the text using the [within helper](https://callstack.github.io/react-native-testing-library/docs/api#within-getqueriesforelement) to limit the scope to the Privacy and Rating screen.

I considered this when putting together the PR, but made edits in other files to get this to work (as "Privacy and Rating" also appears multiple times within the component). That's what led me to believe this approach was the most straightforward option. Am I missing another way to achieve this perhaps?

Reply:
This approach is great to me, I just wanted to note that using the [`within` helper](https://callstack.github.io/react-native-testing-library/docs/api#within-getqueriesforelement) might help in case we'd like to query elements that display value is contained multiple times within the content.

Reply:
Thanks Carlos! If I'm missing a way to use it here, let me know, otherwise I'll bear this in mind for the future 🙇‍♀️